Directions Example/MBDrawingView.swift

```diff
@@ -1,95 +0,0 @@
-import UIKit
-
-public protocol MBDrawingViewDelegate: class {
-    func drawingView(drawingView: MBDrawingView, didDrawWithPoints points: [CGPoint])
-}
-
-public class MBDrawingView: UIView {
```

Comment:
For context, [MBDrawingView.swift](https://github.com/mapbox/MBDrawingView.swift/) was used to demonstrate integration with the Map Matching API. I think it’s OK to remove this particular demonstration now.

Reply:
I wasn't sure what it did. Thanks for the clarification. 
I will add this point to the list of things to think about. Removing dependency on Maps SDK have the benefits of simplifying this repo. Maybe we can have a more sophisticated example in a separate repo similar to how we have https://github.com/mapbox/mapbox-navigation-ios-examples